src/features/common/types/user.d.ts

```diff
@@ -17,3 +18,33 @@ export interface PublicUser {
   /** @deprecated do not use */
   hasLogoLicense?: boolean;
 }
+
+export interface ContributionType {
+  type: string;
+  geometry: Geometry;
+  properties: Properties;
+}
+interface Properties {
+  id: string;
+  type: string;
+  plantDate?: string;
+  country: string;
+  images: Image[];
+  treeCount: number;
+  unitType: string;
+  project: Project;
+  recipient?: Owner | null;
+  giver?: Owner | null;
+}
```

Comment:
I'd suggest a small renaming here. `ContributionItem` in place of `ContributionType`, and `ContributionProperties` instead of `Properties` as it is too generic.